json-parser/index.hs

```diff
@@ -0,0 +1,255 @@
+-- # A JSON parser in Haskell
+--
+-- **License:** MIT
```

Comment:
License specified here.

Reply:
Oh, sorry, not sure how I overlooked this 😬